lib/engine/game/g_18_chesapeake.rb

```diff
@@ -22,13 +22,13 @@ class G18Chesapeake < Base
           %w[B6 B8 B10 C3 C7 C9 C11 E7 E9 E13 F6 F12 G7 I7 J8 J10 L4 F4 G5 H2 I3] => 'blank',
           %w[B12 D4 D6 D10 E5] => 'mtn80',
           %w[F10 G9 G11 H12] => 'wtr40',
-          %w[B4 K3 K5] => 't=r:0;t=r:0',
+          %w[B4 K3 K5] => 'double_town',
```

Comment:
i think we're trying to move away from these aliases, thoughts @kelsin

Reply:
If that's the way we want to go, I'll update the PR to ditch all of these aliases.

Reply:
@kelsin thoughts?